src/test/java/com/atlassian/db/replica/internal/ExperimentsTest.java

```diff
@@ -0,0 +1,44 @@
+package com.atlassian.db.replica.internal;
+
+import org.assertj.core.api.Assertions;
+import org.junit.Test;
+
+import java.util.Collections;
+
+import static com.google.common.collect.ImmutableList.of;
+
+public class ExperimentsTest {
+
+    @Test
+    public void shouldBeEnabledByDefault() {
```

Comment:
For me is's unnatural. Experiment should be disabled by default, as its experiment

Reply:
This is a library. We shouldn't force all users of the library to use experiments API. When you bump a library, your default expectation is to see the effect of changes or to have to find an experiment's key and switch it?